script.js

```
@@ -0,0 +1,25 @@
+function daysInYear (year) {
+  const promise = new Promise((resolve, reject) => {
+    if (typeof(year) === 'number' && Number.isInteger(year)){
+      resolve(year)
+    } else {
+      reject(new Error('exception'))
+    };
+  });
+
+  promise
+    .then(
+      year => { 
+        year % 400 === 0 || !(year % 100 === 0 || !(year % 4 === 0)) ? console.log(`in ${year} year 366 days`) : console.log(`in ${year} year 365 days`);
+      }
+    )
+    .catch(error => console.log(error));
```

This function should throw an exception instead of writing to console.